tools/update_cpu_features.zig

```diff
@@ -1013,12 +1013,6 @@ const llvm_targets = [_]LlvmTarget{
         .llvm_name = "Sparc",
         .td_name = "Sparc.td",
     },
-    // TODO: merge tools/update_spirv_features.zig into this script
-    //.{
-    //    .zig_name = "spirv",
-    //    .llvm_name = "SPIRV",
-    //    .td_name = "SPIRV.td",
-    //},
```

Comment:
Maybe we need something equivalent to this again with LLVM 20? Since SPIR-V is enabled by default in LLVM 20?

Reply:
Do we actually want to enable using LLVM's SPIR-V backend with Zig? I'd brought that up a while back (I think around the time of the LLVM 19 upgrade?) and there seemed to be some resistance to the idea.

Right now, we explicitly prevent it:

https://github.com/ziglang/zig/blob/8a3aebaee0a68d037a6f311bc5c1b426e8e1884c/src/codegen/llvm.zig#L13019-L13023

Reply:
We can't do that yet because [they define no features](https://github.com/llvm/llvm-project/blob/5c93eb56dc9bc0c0210483cdd5d31e6b6580454f/llvm/lib/Target/SPIRV/SPIRV.td)

Reply:
> Do we actually want to enable using LLVM's SPIR-V backend with Zig? I'd brought that up a while back (I think around the time of the LLVM 19 upgrade?) and there seemed to be some resistance to the idea.

Since its now being enabled by default, I don't see why not. It should be a minor addition in theory...